src/commands/generate.rs

```diff
@@ -36,3 +37,10 @@ fn command(name: &str, binary_path: PathBuf, args: &[&str]) -> Command {
     c.args(args);
     c
 }
+
+fn project_type(template: &str) -> ProjectType {
+    if template.contains("rust") {
+        return ProjectType::Rust;
+    }
+    ProjectType::JavaScript
```

Comment:
Also consider:

```rust
if template == "rust" {
    ProjectType::Rust
} else {
    ProjectType::JavaScript
}
```

Though this is of course subjective

Reply:
yeah i prefer it the way it currently is :)